CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,35 @@
+### CODE OF CONDUCT
+Express Scripts Strategic Development, Inc. (“Express Scripts”) welcomes contributions to our hosted open source projects.  We have adopted this Code of Conduct (the “Code”) to facilitate your participation and to ensure our open source community remains a safe, harassment-free, and collaborative space. 
+
+# Scope
+This Code applies within all community spaces, and also applies when an individual is officially representing the community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+# Our Standards
+We strive to make participation in our open source projects open and enjoyable for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex, gender identity, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual orientation.  
+
+Examples of behavior that contributes to a positive environment for our community include:
+* Demonstrating empathy and kindness toward other people.
+* Being respectful of differing opinions, viewpoints, and experiences.
+* Giving and gracefully accepting constructive feedback.
+* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience.
+* Focusing on what is best not just for us as individuals, but for the overall community.
+
+Examples of unacceptable behavior include:
+* The use of sexualized language or imagery, and sexual attention or advances of any kind.
+* Trolling, insulting or derogatory comments, and personal or political attacks.
+* Public or private harassment.
+* Publishing others’ private information, such as a physical or email address, without their explicit permission.
+* Other conduct which could reasonably be considered inappropriate in a professional setting.
+
+# Enforcement 
+The Express Scripts Open Source Leadership Group (the “Leadership Group”) is responsible for clarifying and enforcing this Code and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+The Leadership Group has the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code, and will communicate reasons for moderation decisions when appropriate.
+
+#Reporting 
```

Comment:
Please add a space between # and Reporting.

Reply:
done.